tests/testthat/setup-fakedata.R

```diff
@@ -1,9 +1,8 @@
 ## Loading relevant libraries for tests
 library(testthat)
-library(xml2)
-library(tibble)
+require(tibble, quietly = TRUE)
 library(rtables)
```

Comment:
This is not needed as `testthat` automatically attach tested package namespace (including private functions)
```suggestion
```